src/main/java/uk/gov/hmcts/reform/prl/services/SolicitorEmailService.java

```diff
@@ -94,6 +94,18 @@ public void sendEmail(CaseDetails caseDetails) {
 
     }
 
+    public void sendReSubmitEmail(CaseDetails caseDetails) {
+        log.info("Sending case resubmission email to solicitor for caseId {}", caseDetails.getId());
+        String applicantSolicitorEmailAddress = caseDetails.getData().get("applicantSolicitorEmailAddress").toString();
```

Comment:
its good to define this as constant, as this string literal is being used at multiple places

Reply:
Added constant to address this.